eclipse-jdt/eclipse.jdt.core
# Parser changes for JEP 443

## What it does
Support parsing unnamed variables and unnamed patterns.

## How to test
Create a Java 21 project with preview features enabled. Ensure that the parser can parse unnamed variables and unnamed patterns as described in https://openjdk.org/jeps/443.

## Author checklist
